 active-rfcs/0001-issue-helper.md

```diff
@@ -0,0 +1,105 @@
+    url: https://prow.tidb.io/tichi/new-issue
+    about: Please use the following link to create a new issue.
+```
+
+The effect of using this feature can be seen in the [vuejs repository](https://github.com/vuejs/vue/issues/new/choose).
+
+## External contact links combined with GitHub Issues
+
+Once we enable external links, we can direct the user or contributor to a web page that we've developed, where we'll use a standard form for the user or contributor to fill out the report, and then we'll use the [GitHub Issue creation API](https://docs.github.com/en/github/managing-your-work-on-github/about-automation-for-issues-and-pull-requests-with-query-parameters#supported-query-parameters) to resubmit the form to GitHub and create the Issue.
+
+The API looks like this: https://github.com/pingcap/tidb/issues/new?title=example&body=FormContent
+
+## Issue Helper
+
+The main issues we need to focus on in TiDB are divided into two categories, one is [Bug Report](https://github.com/pingcap/tidb/blob/master/.github/ISSUE_TEMPLATE/bug-report.md) and the other is [New Feature Request](https://github.com/pingcap/tidb/blob/master/.github/ISSUE_TEMPLATE/feature-request.md).
+
+So our Issue Helper focuses on these two types of reports and develops forms for them and then fills them back into the GitHub Issue.
+
+### Bug Report Template
+
+```md
+### Steps to reproduce
+
+### What is expected?
+
+### What is actually happening?
+
+### Any additional comments? (optional)
+
+| Environment   | Info   |
+| ------------- | ------ |
+| TiDB Version  | v4.0.1 |
+| MySQL Version | 8.0.19 |
+
+<!-- generated by issue-helper. DO NOT REMOVE -->
+```
+
+### Feature Request Template
+
+```md
+### What problem does this feature solve?
+
+### What does the proposed feature look like?
+
+### Any additional comments? (optional)
+
+<!-- generated by issue-helper. DO NOT REMOVE -->
+```
+
+We will develop the form based on the above two templates, and all the above questions will be filled in as a field by the user or contributor in the page we develop.
+
+In addition, we need to support auto-completion and search for related issues that have been created when a user or contributor fills in the issue title. This feature will use [GitHub's search API](https://docs.github.com/en/rest/reference/search#search-issues-and-pull-requests).
+
+For specific page style implementations, we can refer to the following two examples:
+
+- [Vue Issue Helper](https://new-issue.vuejs.org/?repo=vuejs/vue)
+- [Ant Design Issue Helper](https://new-issue.ant.design/)
+
+# Drawbacks
+
+Using a web page for users or contributors to fill out the report is simpler and easier than using a complex markdown to fill out as a template. So I see no drawback to this solution.
+
+# Alternatives
+
+N/A
+
+# Adoption strategy
+
+We can use the tool to create a new Issue without modifying the one we have already created.
+
+# Unresolved questions
+
+- Should we allow Issue creation without using this page?
```

Comment:
Under what circumstances do users need other ways to create issues, can it be solved by `### Any additional comments? (optional)`?
In addition, we can also define the availability of issue helper in this RFC, such as guaranteeing issue helper sla 99.9%, how about it?

Reply:
> Under what circumstances do users need other ways to create issues, can it be solved by `### Any additional comments? (optional)`?

Since there are some other templates in the tidb repository, I think we should let people continue to use the others.

See also: https://github.com/ti-community-infra/rfcs/pull/3#discussion_r580821536